### PR TITLE
feat(data-explorer): Add placeholders

### PR DESCRIPTION
**Pull request by MaxPostema**
You can test with: 
http://localhost:8080/entity/it_emx_datatypes_TypeTest
This will give some v3 test data without molgenis connection

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Conventional commits (squash if needed)
- [x] No warnings during install
- [x] Updated javascript typing

### CODE REVIEW

File: packages/data-explorer/src/components/ToolbarView/ActiveFilters.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+library.add(faTimes)
+
+export default Vue.extend({
+  name: 'ActiveFilters',
+  components: { FontAwesomeIcon }
+})
+</script>
+
+<style scoped>
+  button svg path{
```
Comment (marikaris) [inline packages/data-explorer/src/components/ToolbarView/ActiveFilters.vue]: space between path and {

File: packages/data-explorer/src/components/ToolbarView/ActiveFilters.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    transition: fill 0.3s;
+  }
+  button:hover svg path{
```
Comment (marikaris) [inline packages/data-explorer/src/components/ToolbarView/ActiveFilters.vue]: space between path and {

File: packages/data-explorer/src/components/Utils/InfoIconButton.vue [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  }
+})
+</script>
+
+<style scoped>
+  button{
+    vertical-align: middle;
+    overflow: hidden;
+  }
+  button .name{
```
Comment (marikaris) [inline packages/data-explorer/src/components/Utils/InfoIconButton.vue]: space between path and {

File: packages/data-explorer/src/components/Utils/InfoIconButton.vue [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    margin: 0 !important;
+    opacity: 0;
+    max-width: 0;
+    display: inline-block;
+    white-space: nowrap;
+    transition: opacity 0.3s, max-width 0.3s, margin 0.3s;
+  }
+  button:hover .name{
```
Comment (marikaris) [inline packages/data-explorer/src/components/Utils/InfoIconButton.vue]: space between path and {

File: packages/data-explorer/src/components/Utils/InfoIconButton.vue [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+})
+</script>
+
+<style scoped>
+  button{
+    vertical-align: middle;
+    overflow: hidden;
+  }
+  button .name{
+    margin: 0 !important;
```
Comment (marikaris) [inline packages/data-explorer/src/components/Utils/InfoIconButton.vue]: Please check if we really need this !important

File: packages/data-explorer/src/components/Utils/InfoIconButton.vue [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    opacity: 0;
+    max-width: 0;
+    display: inline-block;
+    white-space: nowrap;
+    transition: opacity 0.3s, max-width 0.3s, margin 0.3s;
+  }
+  button:hover .name{
+    margin-left: 0.5rem !important;
```
Comment (marikaris) [inline packages/data-explorer/src/components/Utils/InfoIconButton.vue]: Please check if we really need this !important

File: packages/data-explorer/src/components/Utils/InfoIconButton.vue [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    max-width: 200px;
```
Comment (marikaris) [inline packages/data-explorer/src/components/Utils/InfoIconButton.vue]: Maybe use %, em or rem?

File: packages/data-explorer/src/store/actions.ts [hunk 8]
```diff
@@ -0,0 +1,10 @@
+// @ts-ignore
+import api from '@molgenis/molgenis-api-client'
+import { tryAction } from './helpers'
+
+export default {
+  loadEntity: tryAction(async ({ commit, state } : any) => {
+    const response = await api.get('/api/entity/' + state.activeEntity)
```
Comment (marikaris) [inline packages/data-explorer/src/store/actions.ts]: I always find the ES6 way of writing this prettier:
```api.get(`/api/entity/${state.activeEntity}`)```

File: packages/data-explorer/src/views/MainView.vue [hunk 9]
```diff
@@ -1,23 +1,80 @@
...
+  },
+  created () {
+    this.$store.commit('setActiveEntity', this.$route.params.entity)
+  },
+  components: { FiltersView, DataView, ToastComponent }
 })
 </script>
+
+<style scoped>
+  .flex-mainview{
```
Comment (marikaris) [inline packages/data-explorer/src/views/MainView.vue]: Please add space

File: packages/data-explorer/src/views/MainView.vue [hunk 10]
```diff
@@ -1,23 +1,80 @@
...
+    display: flex;
+    white-space: normal;
+  }
+  .flex-filter{
```
Comment (marikaris) [inline packages/data-explorer/src/views/MainView.vue]: Please add space

File: packages/data-explorer/src/views/MainView.vue [hunk 11]
```diff
@@ -1,23 +1,80 @@
...
+    transition: max-width 0.3s, min-width 0.3s;
+    min-width: 20rem;
+    max-width: 20rem;
+    padding-right: 1rem;
+    flex: 1 0 0rem;
+    overflow: hidden;
+
+  }
+  .showfilters .flex-filter{
```
Comment (marikaris) [inline packages/data-explorer/src/views/MainView.vue]: Please add space

File: packages/data-explorer/src/views/MainView.vue [hunk 12]
```diff
@@ -1,23 +1,80 @@
...
+  }
+  .flex-data{
+    flex: 1 0 0
+  }
+
+  @media only screen and (max-width: 576px) { /* Bootstrap brakepoint sm */
+    .flex-mainview{
+      display: block;
+    }
+    .flex-mainview .flex-filter{
```
Comment (marikaris) [inline packages/data-explorer/src/views/MainView.vue]: Please add space

File: packages/data-explorer/src/views/ToolbarView.vue [hunk 13]
```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="toolbar">
+    <info-icon-button
+      v-if="showFilters === true"
```
Comment (marikaris) [inline packages/data-explorer/src/views/ToolbarView.vue]: isn't this the same as:
```
v-if="showFilters"
```

Comment (MaxPostema) [reply packages/data-explorer/src/views/ToolbarView.vue]: Hmmm... good point.  👍

**Comment by marikaris** [comment]
We have to figure out how to configure sonar to only look at src and test dir, now it looks at config and there code is duplicated because each plugin has its own resources.

**Pull request merged**